ci/tasks/assume-role.yml

```diff
@@ -0,0 +1,36 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: amazon/aws-cli
+    tag: '2.0.44'
+outputs:
+  - name: assume-role
+params:
+  AWS_ROLE_ARN:
+  AWS_ROLE_SESSION_NAME:
+run:
+  path: sh
+  args:
+    - -c
+    - |
+      set -eu
+      aws sts assume-role \
+        --role-arn "${AWS_ROLE_ARN}" \
+        --role-session-name "${AWS_ROLE_SESSION_NAME}" \
+      | python -c '
```

Comment:
cheeky bit of python 🐍

Reply:
this is very much copy-pasta (I think originally from young Toby). It's basically either that or bash or make a new container (although thinking about it we now have a new container node aws container so maybe we should use that)

Reply:
in fact I think we definitely should do that. Maybe in a separate PR